types/twitter-text/index.d.ts

```diff
@@ -112,3 +112,15 @@ export declare function getUnicodeTextLength(text: string): number;
 export declare function convertUnicodeIndices(text: string, entities: EntityWithIndices[], indicesInUTF16?: boolean): void;
 
 export declare function hitHighlight(text: string, hits?: number[][], options?: { tag: string }): string;
+
+export interface ParsedTweet {
+  weightedLength: number;
+  permillage: number;
+  valid: boolean;
+  displayRangeEnd: number;
+  displayRangeStart: number;
+  validRangeEnd: number;
+  validRangeStart: number;
+}
+
+export declare function parseTweet(text: string): ParsedTweet;
```

Comment:
It takes second parameter. Please consider it. https://github.com/twitter/twitter-text/blob/master/js/src/parseTweet.js#L23

Reply:
Done